application.py

```diff
@@ -1,25 +1,28 @@
 import requests
 import json
+import allure
+
 from base import LOGIN_URL, BASE_URL
 
 HEADERS = {'content-type': 'application/json'}
 
+
 class Application():
     """The parent class for all tests"""
 
     def __init__(self):
         self.session = requests.Session()
 
     def authentication(self, username, password):
-        self.request = self.session.post(LOGIN_URL, auth=(username, password))
-        if self.request.status_code == 200:
+        request = self.session.post(LOGIN_URL, auth=(username, password))
+        if request.status_code == 200:
             self.get(BASE_URL)
             self.session.headers.update({"X-XSRF-TOKEN": self.session.cookies.get_dict()['XSRF-TOKEN'],
```

Comment:
how about making "X-XSRF-TOKEN" and "XSRF-TOKEN" as constants?

Reply:
they change each time when tests start.